packages/liveblocks-core/src/ai.ts

```diff
@@ -1144,9 +1145,16 @@ export function createAi(config: AiConfig): Ai {
     }
   });
 
+  function connectInitially() {
+    if (managedSocket.getStatus() === "initial") {
```

Comment:
I initially did that (directly in `useEnsureAiConnection`) but thought the state machine handled not reconnecting automatically, was the issue that it's not the case?